main.cpp

```diff
@@ -1529,133 +1285,143 @@ VOID SaveSettings(VOID)
 			LocalFree(blobOut.pbData);
 		}
 
-		RegCloseKey (hk);
+		RegCloseKey(hk);
 	}
 }
-
+
 LRESULT CALLBACK ProcMain(HWND hDlg, UINT message, UINT wParam, LONG lParam)
 {
 	switch (message)
 	{
-		case WM_INITDIALOG:
-			ProcMainInit(hDlg);
-			return FALSE;
+	case WM_INITDIALOG:
+		ProcMainInit(hDlg);
+		return FALSE;
 
 		//case WM_QUERYUISTATE:
 			//return UISF_HIDEFOCUS;
 
-		case WM_CHANGEUISTATE:
+	case WM_CHANGEUISTATE:
+	{
+		// Disable focus rectangles by setting or masking out the flag where appropriate. 
+		switch (LOWORD(wParam))
 		{
-			// Disable focus rectangles by setting or masking out the flag where appropriate. 
-			switch (LOWORD(wParam))
-			{
-			case UIS_SET:
-				wParam |= UISF_HIDEFOCUS << 16;
-				break;
-			case UIS_CLEAR:
-			case UIS_INITIALIZE:
-				wParam &= ~(UISF_HIDEFOCUS << 16);
-				break;
-			}
+		case UIS_SET:
+			wParam |= UISF_HIDEFOCUS << 16;
+			break;
+		case UIS_CLEAR:
+		case UIS_INITIALIZE:
+			wParam &= ~(UISF_HIDEFOCUS << 16);
+			break;
 		}
-		break;
-
-		case WM_COMMAND :
-			switch (LOWORD(wParam))
+	}
+	break;
+	case WM_COMMAND:
+		switch (LOWORD(wParam))
+		{
+			//A command was fired, which one?
+		case IDC_US:

```

Comment:
here we check if the user has clicked one of the boxes, and run the function to populate the list.